src/i18n/ar.json

```diff
@@ -89,6 +89,7 @@
     "getStarted": "البدء!",
     "addFirstTime": "مرحبا ، أضف قائمتك الأولى!",
     "sortByName": "THE",
+    "sortByProgress": "Progress",
```

Comment:
you can run `yarn translate` to translate all the strings (lmk if you need an API key for google translate, or you can get a free one)

Reply:
I'd need an API key. I'm having trouble navigating Google's developer documentation.

Reply:
Cool, just emailed you the key 👍